dev/release/verify-release-candidate.sh

```diff
@@ -831,7 +831,7 @@ test_glib() {
   show_header "Build and test C GLib libraries"
 
   # Build and test C GLib
-  maybe_setup_conda glib gobject-introspection meson ninja ruby
+  maybe_setup_conda glib==2.80.5 gobject-introspection meson ninja ruby
```

Comment:
```suggestion
  # We can remove '==2.80.5' once https://github.com/conda-forge/glib-feedstock/issues/191
  # is fixed.
  maybe_setup_conda glib==2.80.5 gobject-introspection meson ninja ruby
```

Reply:
👍 thank you! Should've done this originally.